### PR TITLE
Add ButtonGroup story for Storybook

### PR DESCRIPTION
**Pull request by mkaz**
## Description

Adds ButtonGroup to StoryBook

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate.

### CODE REVIEW

**Review by epiqueras — APPROVED** [review]

**Comment by epiqueras** [comment]
Oops, I missed the "In Progress" label, but the changes are solid and we can work on top of them :smile:

**Comment by mkaz** [comment]
@epiqueras Not a problem, I only put in progress because I wasn't sure if it was all that was needed. :+1:

**Comment by epiqueras** [comment]
We could add more variations with different numbers of buttons.

File: packages/components/src/button-group/stories/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ */
+import Button from '../../button';
+import ButtonGroup from '../';
+
+export default { title: 'Button Group', component: ButtonGroup };
+
+export const _default = () => (
+	<ButtonGroup>
+		<Button isPrimary>Button 1</Button>
+		<Button isPrimary>Button 2</Button>
```
Comment (gziolo) [inline packages/components/src/button-group/stories/index.js]: I reason that only one of them should be primary:

Screen Shot 2019-10-11 at 12 51 10

They look like one big button at the moment.

Comment (mkaz) [reply packages/components/src/button-group/stories/index.js]: Is there a way to add CSS to Storybook? Also, the buttons when running in dev look different to  than the buttons in production, which makes me think there is CSS loaded somehow

Comment (epiqueras) [reply packages/components/src/button-group/stories/index.js]: https://storybook.js.org/docs/configurations/default-config/#css-support

Comment (mkaz) [reply packages/components/src/button-group/stories/index.js]: Addressed in #17910

**Pull request merged**